packages/now-cli/src/commands/dns/add.ts

```diff
@@ -54,9 +56,23 @@ export default async function add(
     );
     return 1;
   }
+ 
+  const { domain, data: argData } = parsedParams;
+  if (argData && domain === argData.name.substr(argData.name.length - domain.length, argData.name.length)) {
```

Comment:
Is this the same logic?

```suggestion
  if (argData && argData.name.endsWith(domain)) {
```

Reply:
Nice catch!